### PR TITLE
Fixed #5418 -- Check plugin permissions when clearing placeholder.

### LINKED ISSUE #5418: Clearing placeholder does not check plugin permissions
Currently a user can clear a placeholder as long as the user has page permissions, regardless of the plugin permissions.

### PR DESCRIPTION
**Pull request by czpython**
Fixes #5418

Note that because this is a bugfix, I've opted to keep the impact as small as possible and thus
did not change much of the internals.

A refactor will come soon..

### CODE REVIEW

**Comment by czpython** [comment]
Please review @mkoistinen @yakky

File: cms/models/placeholdermodel.py [hunk 1]
```diff
@@ -157,6 +158,21 @@ def has_add_permission(self, request):
...
 
+    def has_clear_permission(self, user, language):
+        plugin_types = (
+            self
+            .get_plugins(language)
+            .values_list('plugin_type', flat=True)
+            .distinct()
+        )
+
+        has_permission = permissions.has_plugin_permission
```
Comment (mkoistinen) [inline cms/models/placeholdermodel.py]: Nice touch =)

**Comment by mkoistinen** [comment]
This really is a light-touch. I guess more of that "string" is left for later releases. Nice looking overall! OK to merge.

**Comment by yakky** [comment]
LGTM

**Pull request closed without merging**